src/printer.js

```diff
@@ -166,14 +167,25 @@ function printExpression(path, options, print) {
         return concat([node.type + node.type, path.call(print, "what")]);
       case "post":
         return concat([path.call(print, "what"), node.type + node.type]);
-      case "bin":
+      case "bin": {
+        let type = node.type;
+
+        if (type.toLowerCase() === "and") {
+          type = "&&";
+        }
+
+        if (type.toLowerCase() === "or") {
+          type = "||";
+        }
```

Comment:
More pretty

Reply:
Does this also handle the difference between `||` and `or`? (https://stackoverflow.com/questions/17710924/logical-operators-or-vs-double-pipe-in-php)

Reply:
:+1: